### PR TITLE
Correction of Clone Repository Bug

### PR DESCRIPTION
**Pull request by CalebeMiranda**
Remove the space between config_/agents.yaml and tasks.yaml. (Represented with "_")
To prevent error: invalid path 'screenplay_writer/config /agents.yaml' and error: invalid path 'screenplay_writer/config /tasks.yaml' when cloning the repository.

### CODE REVIEW

**Review by theCyberTech — CHANGES_REQUESTED** [review]
Can you please fix up the space in front of agents.yml then I can merge

File: screenplay_writer/config/ agents.yaml
Comment (agrajagco) [inline screenplay_writer/config/ agents.yaml:1]: It appears there is still a space in front of the agents.yaml @CalebeMiranda

**Comment by agrajagco** [comment]
I've created a corrective PR request, I've tagged @CalebeMiranda as a co-author

**Comment by theCyberTech** [comment]
fixed in #157

**Pull request closed without merging**